target/cheri-common/cheri-lazy-capregs-types.h

```diff
@@ -69,13 +69,11 @@ typedef enum CapRegState {
 
 // Cap registers should be padded so they are easier to move.
 #if TARGET_LONG_BITS == 32
-_Static_assert(sizeof(cap_register_t) == 40, "");
-#else
-_Static_assert(sizeof(cap_register_t) == 64, "");
```

Comment:
Oh, hm, I imagine we want the CHERI-128 one too?

Reply:
Yes we do, not sure how I dropped that from this commit..